distro/README.md

```diff
@@ -52,19 +56,36 @@ cd release-from-archive
 tar xzf ../riff-release.tar.gz
 ```
 
-Relocate the images in the release into the `relocated` directory, specifying a new `dev.local` registry and `u` user, and rewriting all the release yaml with the relocated image names/tags. Using `--flatten` is required for registries which don't support deep hierarchies of image names.
+Relocate the images into a `relocated` directory, specifying the `registry` and `user-repo`. This will create a new manifest and new release yaml files, prefixing all the image names with `registry`/`user-repo`.
+
 ```sh
-riff image relocate -m manifest.yaml -r dev.local -u u -i image-manifest.yaml -o relocated --flatten
+export REGISTRY_NAME=docker.io  # replace with your private registry name
+export REGISTRY_ID=???          # replace with your registry repo/account
+
+riff image relocate \
+  -m manifest.yaml \
+  -r $REGISTRY_NAME \
+  -u $REGISTRY_ID \
+  -i image-manifest.yaml \
+  -o relocated \
+  --flatten
 ```
 
-Load the relocated images from the file system into the registry.
+Using `--flatten` is required for registries which don't support deep hierarchies of image names like Docker.
+
+Push the relocated images to a registry. This first loads the images from the file system and tags them in your local docker daemon.
+The final push step requires that docker has push credentials for the registry.
 ```sh
 cd relocated
-riff image load -i image-manifest.yaml
+riff image push -i image-manifest.yaml
 ```
 
-Install riff and initialize the default namespace using relocated images.
+To install riff locally (e.g. for minikube or Docker Desktop), use `riff image load` instead of `riff image push`.
+This loads and tags images locally without pushing them to a registry.
+
+Install riff and initialize the default namespace using the relocated manifest and images.
 ```sh
 riff system install -m manifest.yaml --node-port
```

Comment:
Shouldn't those lines clearly state that `manifest.yaml` is the "relocated" one? eg maybe using `relocated/manifest.yaml`?

Reply:
Stating it in both the sentence and the command can't hurt